apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -151,39 +258,175 @@ class TeacherFeedback extends Component {
             {i18n.feedbackLoadError()}
           </span>
         )}
-        <textarea
-          id="ui-test-feedback-input"
-          style={styles.textInput}
-          onChange={this.onCommentChange}
-          placeholder={placeholderText}
-          value={this.state.comment}
-        />
-        <div style={styles.footer}>
-          <div style={styles.button}>
-            <Button
-              id="ui-test-submit-feedback"
-              text={buttonText}
-              onClick={this.onSubmitFeedback}
-              color={Button.ButtonColor.blue}
-              disabled={buttonDisabled}
-            />
-            {this.state.errorState === ErrorType.Save && (
-              <span>
-                <i className="fa fa-warning" style={styles.errorIcon} />
-                {i18n.feedbackSaveError()}
-              </span>
-            )}
+        {this.props.rubric && (
+          <div style={styles.performanceArea}>
+            <div style={styles.keyConceptArea}>
+              <h1 style={styles.h1}>Key Concepts</h1>
+              <p style={styles.keyConcepts}>{this.props.rubric.keyConcept}</p>
+            </div>
+            <div style={styles.rubricArea}>
+              <h1 style={styles.h1}>Evaluation Rubric</h1>
+              <form>
+                <div
+                  style={
+                    this.state.performance === 'exceeds'
+                      ? styles.performanceLevelHeaderSelected
+                      : styles.performanceLevelHeader
+                  }
+                >
+                  {showFeedbackInputAreas && (
+                    <input
+                      type={'checkbox'}
+                      id={'exceeds-input'}
+                      name={'rubric'}
+                      value={'exceeds'}
+                      checked={this.state.performance === 'exceeds'}
+                      onChange={this.onRubricChange}
+                      disabled={this.props.disabledMode}
+                      style={styles.checkbox}
+                    />
+                  )}
+                  <details>
+                    <summary style={styles.rubricHeader}>Exceeds</summary>
+                    <p>{this.props.rubric.exceeds}</p>
+                  </details>
+                </div>
```

Comment:
this render method is getting a bit long. if it would be ideal if you could extract a subcomponent. I think you could extract everything in this div, and pass in just the string 'exceeds', 'meets', etc to parameterize it -- but this could be a bit of a project, so it would be ok to wait until a later PR as this one is already quite big.

Reply:
PR for this: https://github.com/code-dot-org/code-dot-org/pull/27175